test/unit/test/streaming/streaming.models.CmcdModel.js

```diff
@@ -425,7 +425,7 @@ describe('CmcdModel', function () {
                     return e.split('=')[0]
                 })).to.not.include('dl');
                 expect(headers[STATUS_HEADER_NAME]).to.be.undefined;
-                expect(headers[SESSION_HEADER_NAME]).to.be.empty;
+                expect(headers[SESSION_HEADER_NAME]).to.be.undefined;
```

Comment:
CML was updated to always exclude header fields with empty values, meaning header shards with no CMCD data will always be `undefined`, not an empty string.